sdks/python/apache_beam/runners/common.py

```diff
@@ -546,7 +551,12 @@ def process_outputs(self, windowed_input_element, results):
 class _OutputProcessor(OutputProcessor):
   """Processes output produced by DoFn method invocations."""
 
-  def __init__(self, window_fn, main_receivers, tagged_receivers):
+  def __init__(self,
+               window_fn,
+               main_receivers,
+               tagged_receivers,
+               counter_factory=None,
+               operation_name=None):
```

Comment:
Can you please create the counter in `DoFnRunner.__init__` and pass the output counter into this method instead of leaving the logic to create the counter in this method?

Also, please don't make the added argument optional--this file is the only place the `_OutputProcessor` is referenced.

Reply:
will fix in next revision